src/components/Fields/TextArea.js

```diff
@@ -262,38 +270,49 @@ class TextArea extends Component {
                 style={this.props.style}
                 {...proxyDataProps(this.props)}
             >
-                <div className="uir-text-area-inner">
-                    <div className="uir-text-area-label-wrapper">
-                        {label}
+                <div>
```

Comment:
Is this outer div needed?

Reply:
Yep. I hat do keep it to put the validation message outside the input field.

![image](https://user-images.githubusercontent.com/348776/68605268-e0b24980-048a-11ea-956f-70734149743b.png)